cmd/podman/auto-update.go

```diff
@@ -49,11 +49,36 @@ func autoUpdate(cmd *cobra.Command, args []string) error {
 		// Backwards compat. System tests expect this error string.
 		return errors.Errorf("`%s` takes no arguments", cmd.CommandPath())
 	}
+
 	report, failures := registry.ContainerEngine().AutoUpdate(registry.GetContext(), autoUpdateOptions)
-	if report != nil {
-		for _, unit := range report.Units {
-			fmt.Println(unit)
+	if report == nil {
+		return errorhandling.JoinErrors(failures)
+	}
+
+	if len(report.Containers) > 0 {
+		fmt.Println("Scanned the following containers:")
+		for _, c := range report.Containers {
+			fmt.Printf(" * ID: %s, name: %s, image: %s, unit: %s, updated: %v", c.ID[:12], c.Name, c.Image, c.SystemdUnit, c.Updated)
```

Comment:
Could we do table output on this and Images? I don't know if our table writer supports doing more than one, but it seems like it would be a lot cleaner, output-wise.

Reply:
Maybe we can squeeze everything into one table:
```
UNIT               CONTAINER         IMAGE            UPDATED        PENDING-UPDATE
unit-name          container-name    raw-image-name   {false,true}   {false,true}
```

While the anticipated `--json` would just spit out all data as raw JSON (including digests and IDs etc).

What do you think?

Reply:
I think that's probably a better solution - very obvious what happened when it's printed this way